### PR TITLE
Added social links to main page

### PR DESCRIPTION
**Pull request by JFelz**
The images are temporary. I just wanted to add the functionality. I can create a more BTT-style image link for each one now.

### CODE REVIEW

**Comment by JFelz** [comment]
Working on the Social Link graphics, will add those .png files in the folder in the next Pull Request.

File: index.html [hunk 1]
```diff
@@ -111,12 +111,33 @@
             allowfullscreen></iframe>
 
         <a href="" class="getstarted">Coming soon</a>
-        <div>
+        <div class="AboutUs">
```
Comment (jjsub) [inline index.html]: Remove the code of about from this PR

**Review by jjsub — CHANGES_REQUESTED** [review]

File: style.css [hunk 2]
```diff
@@ -152,4 +152,46 @@ body{
...
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    padding-top: 15px;
+}
+
+.insta-link{
+    padding: 5px;
```
Comment (jjsub) [inline style.css]: Also primo use `em` and not px 

This is better for different screen resolutions and sizes

**Comment by jjsub** [comment]
@JFelz Could you please provide screenshots of the UI work related to this PR 

One screenshot for desktop view and one screenshot in mobile view. 

Thanks

**Comment by jjsub** [comment]
@JFelz How is your timing on the updates? 

Would you like me to merge your chanes and then I make the changes myself. If that's okay with you I can go ahead and do that. 

Waiting for you res

**Comment by JFelz** [comment]
Screenshot in mobile view :
image

Screenshot in Desktop view:
image

**Review by JFelz — COMMENTED** [review]
Updated the measurement types and removed the about section as requested

**Pull request merged**